src/shared/components/ShareWithColleagues/ShareWithColleagues.tsx

```diff
@@ -46,6 +47,8 @@ const ShareWithColleagues: FC<ShareWithColleaguesProps & UserProps> = ({
 	const [error, setError] = useState<string | null>(null);
 	const [isEditRightsModalOpen, setIsEditRightsModalOpen] = useState<boolean>(false);
 	const [toEditShareUser, setToEditShareUser] = useState<ShareUserInfo>();
+	const [isDeleteUserModalOpen, setIsDeleteUserModalOpen] = useState<boolean>(false);
+	const [toDeleteShareUser, setToDeleteShareUser] = useState<ShareUserInfo>();
```

Comment:
I prefer to explicitly say if something can be null or a value. So that typescript warns me if i try to call a property on something that could be null.
I reserve undefined for things that are not set, and were not explicitly set by the developer to null. Like a property that you call that doesn't exist anymore, would return undefined.
```suggestion
	const [toDeleteShareUser, setToDeleteShareUser] = useState<ShareUserInfo | null>(null);
```